dev/release/README.md

```diff
@@ -538,13 +482,14 @@ The Apache Arrow Community
 ### Remove old artifacts from SVN
 
 These artifacts include any release candidates that were uploaded to
-<https://dist.apache.org/repos/dist/dev/arrow/> and old releases that
-were upload to
-<https://dist.apache.org/repos/dist/release/arrow/>. You can remove
-them using:
+<https://dist.apache.org/repos/dist/dev/arrow/>. You can remove them
+using:
 
 ```
-dev/release/post-02-remove-old-artifacts.sh
+# Once
+export APACHE_USERNAME=xxx
+# Once for every release candidate
+svn rm --username=$APACHE_USERNAME -m "Clean up svn artifacts" https://dist.apache.org/repos/dist/dev/arrow/apache-arrow-nanoarrow-0.7.0-rc0/
```

Comment:
@paleolimbot It seems this reverts changes in https://github.com/apache/arrow-nanoarrow/pull/787 . Could you remove old artifacts in https://dist.apache.org/repos/dist/release/arrow/ by `dev/release/post-02-remove-old-artifacts.sh` ?